frontend/src/metabase/writeback/api.ts

```diff
@@ -0,0 +1,3 @@
+import { POST } from "metabase/lib/api";
+
+export const deleteRow = POST("/api/actions/row/delete");
```

Comment:
I guess the preferred way to add new endpoints on FE is to create an API object in `metabase/services`, [see an example](https://github.com/metabase/metabase/blob/9ebcf4b185b91f1031ce40865e460f32350f40ac/frontend/src/metabase/services.js#L138)

Reply:
Yeah that was my first instinct but I figured we'd want to separate out the endpoints so no one else calls them accidentally.

moved to `metabase/services`